0004-rfcs-over-adr-pdr/0004-rfcs-over-adr-pdr.md

```diff
@@ -0,0 +1,9 @@
+# RFC 0004 - RFCs over ADRs and PDRs
+
+This RFC describes the relationship between RFCs, ADRs, and PDRs.
+
+'ADR' refers to Architecture Decision Record, and 'PDR' refers to Product Decision Record. These records, as well as their processes, are currently located in the private Giant Swarm Intranet.
```

Comment:
We explain what ADR and PDR stands for so I'd suggest to also explain what RFC is.